docs/UserGuide.md

```diff
@@ -482,7 +482,7 @@ The fields are:
 - `a/ADDRESS` is the address of the contact.
 - `pr/PRIORITY` is the priority level of the contact.
 - `[c/COMPANY]` (optional) is the index of the company to which the contact belongs.
-- `[t/TAG]` (optional) is used to categorize your contacts. Tags help organize connections efficiently.
+- `[t/TAG]` (optional) is used to categorize your contacts. Multiple Tags can be added by typing a Tag after each `/t` placeholder. Tags help organize connections efficiently.
```

Comment:
Looks good

Reply:
do we need to update this for the other commands that users can include tags? or is this enough